source/connectors/kafka-sink-properties.txt

```diff
@@ -0,0 +1,249 @@
+.. _kafka-sink-properties:
+
+=============================================
+Kafka Sink Connector Configuration Properties
+=============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Sink Connector Configuration Properties
+---------------------------------------
+
+Refer to the table below to create or migrate a properties file for your
+MongoDB Kafka sink connector. See `MongoSinkConnector.properties
+<https://github.com/mongodb/mongo-kafka/blob/master/config/MongoSinkConnector.properties>`_
+for an example configuration file.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 1 1 3
+
+   * - Name
+     - Type
+     - Description
+
+   * - topics
+     - list
+     - | A list of Kafka topics for the sink connector.
+       |
+       | **Default**: (none)
+       | **Accepted Values**: A non-empty list
```

Comment:
1) This should be comma-separated list. Will update this and similar instances.
2) Sure -- everything is required except one optional field, but some properties have default values. Should those also be marked required?
3) I had thought about it, but made the decision to make it symmetrical (since it was previously represented as a cell in a table) and hopefully more clear. Not sure what to do here.

Reply:
Ok, for this document, I will do the following:
- If it **requires users to provide a value**, I'll add "Required" in italics and omit the "Default" field since it's redundant in those cases.
- If a default value exists (users do not have to provide a value, but is required for it to run), I'll add a "Default" field.
- If a default value does not exist and does not require any input (setting is purely optional), or if optional depending on some other parameter, I'll include a "(none)" placeholder in the "Default" field.
- All fields will have a "Valid Values" section

Reply:
I think that approach makes sense. SGTM!